Firebase/Auth/Source/RPCs/FIRAuthBackend.m

```diff
@@ -1069,7 +1076,8 @@ + (nullable NSError *)clientErrorWithServerErrorMessage:(NSString *)serverErrorM
     return [FIRAuthErrorUtils quotaExceededErrorWithMessage:serverErrorMessage];
   }
 
-  if ([shortErrorMessage isEqualToString:kAppNotVerifiedErrorMessage]) {
+  if ([shortErrorMessage isEqualToString:kAppNotVerifiedErrorMessage] ||
```

Comment:
Just to confirm, we want to return an error stating the app isn't verified in the situation where they insert a real phone number but testing is enabled? Wouldn't it make sense to have a specific error for this?

Reply:
Changed the error message for this specific case.